app-text/capyt/capyt-1.1.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5..7} )
+
+inherit distutils-r1
+
+MY_PV=$(ver_rs 2 '-')
+
+DESCRIPTION="A python3 CLI utility to interface with cpy.pt paste service"
+HOMEPAGE="https://github.com/lbatalha/capyt
+	https://pypi.org/project/capyt/"
+SRC_URI="https://github.com/lbatalha/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
https://wiki.gentoo.org/wiki/Project:Python/Dependencies#Dependency_on_setuptools

Reply:
So if I understand this hint correctly, setuptools should be under DEPEND and not BDEPEND?

Reply:
No, DEPEND vs BDEPEND is not covered currently as we don't have cross support.

I meant it's RDEPEND here as well.

Reply:
So, would something like this be correct: 
```
RDEPEND="
    ${PYTHON_DEPS}
    dev-python/requests[${PYTHON_USEDEP}]
    dev-python/setuptools[${PYTHON_USEDEP}]
"
DEPEND="${RDEPEND}"
```

Reply:
I was wondering about PYTHON_DEPS too, 
https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/distutils-r1.eclass#n100

So I can do
```
DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
RDEPEND="dev-python/requests[${PYTHON_USEDEP}]"
```

or

```
RDEPEND="
    dev-python/requests[${PYTHON_USEDEP}]
    dev-python/setuptools[${PYTHON_USEDEP}
"
```

right? Personally prefering the first one.

Reply:
Ok, that's pretty clear. Gonna update and rebase in a minute, thanks for patience and guidance :)

```
DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
RDEPEND="
    ${DEPEND}
    dev-python/requests[${PYTHON_USEDEP}]
"
```